numba/np/arraymath.py

```diff
@@ -3964,6 +3964,10 @@ def histogram_impl(a, bins=10, range=None):
 # This module is imported under the compiler lock which should deal with the
 # lack of thread safety in the warning filter.
 def _gen_np_machar():
+    # NumPy 1.24 removed np.MachAr
+    if numpy_version >= (1, 24):
+        return
```

Comment:
Has this been removed? It looks like it's still in NumPy's source at 1.24 and I couldn't see anything in the release notes about it being removed, but may well have missed something.

Reply:
It has been removed. It still exists in `np.core.MachAr` but that was not the public interface.

Reply:
Thanks for confirming.